### PR TITLE
Test for integration end-to-end

### PR DESCRIPTION
**Pull request by joschkabraun**
WIP on #80

### CODE REVIEW

**Comment by joschkabraun** [comment]
Implemented end-to-end testing for backend with CI

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -97,6 +97,42 @@ jobs:
       - name: Test basic import
         run: python -c 'from now import run_all_k8s'
 
+    integration-test:
```
[the remark this answers is not in the document]
Comment (joschkabraun) [reply .github/workflows/ci.yml:100]: Okay, thank you!
For me there is no need to keep it :)

**Review by makram93 — APPROVED** [review]
LGTM! Nice work!

**Comment by florian-hoenicke** [comment]
@joschkabraun Sorry for the confusion I created. Since the mac runner is 10x expensive, we stay with ubuntu.
Also, if you don't find a way to reduce the runtime, 30min would be fine for me.

**Comment by florian-hoenicke** [comment]
LGTM

**Pull request merged**